app/services/course/assessment/question/programming/cpp/cpp_autograde_pre.cc

```diff
@@ -0,0 +1,132 @@
+// Variant type that encompasses all the types that expect_equal() supports
```

Comment:
I decided to remove the use of the boost library. Initially required it but after changing the structure, i realized that it would not be needed anymore. Would be better also, since it makes things less complicated ><